support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.test.tsx

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable eslint-comments/require-description -- This is a mocks file, it is not intended to be good code! */
 /* eslint-disable @typescript-eslint/no-unsafe-return */
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import '__mocks__/settingsMock';
```

Comment:
I think as we are importing  '__mocks__/settingsMock' to  testStore.ts file, importing it into individual test files may not be necessary

Reply:
Yea, this is true, but if that's the case, I would really like to fund out where the highest entrypoint is that I can do this. Do you have any ideas?

Reply:
We can address this in another PR.